### DISCUSSION TITLE
3D20-Marlin profiles for Cura / Super Slicer, etc?

### DISCUSSION THREAD

**Discussion by bbatchelder** [moonglow/FlashForge_Marlin#66] [General]
Hey folks, I'm curious if anyone has put together slicer profiles for the 3D20 with Marlin firmware loaded?

I've got one started for Cura, based on the 3D20 plugin from Tim (https://github.com/timmehtimmeh/Cura-Dremel-Printer-Plugin)

The biggest change so far being the Start G-code (see wiki warning about not using original flashforge g-code (https://github.com/moonglow/FlashForge_Marlin/wiki/%5BATTENTION%5D-Avoid-using-FlashForge-original-firmware-G-Codes)

**Comment by moonglow** [comment]
Hi !

My main slicer is Cura for my FF Dreamer NX and it works fine for me. Btw with Marlin we can use many other Slicers too with really simple setup.

**Comment by geeeethub** [comment]
@bbatchelder would you be interested in posting your profiles for Cura for the 3D20 w/ Marlin?

**Reply by bbatchelder** [reply]
Absolutely, I just need to clean them up a bit, I did a lot of experimenting coming from Simplify3D and trying to find equivalent settings in Cura.

**Reply by bbatchelder** [reply]
Finally got them to a point I think they're worth sharing....

https://github.com/bbatchelder/cura-dremel-3d20marlin

**Comment by ShinNinth** [comment]
Hey are you still possibly sharing this Cura config? I am not sure other that the M907 code what i should be looking out for for my 3d20 in cura now that I've put marlin on it

**Reply by bbatchelder** [reply]
Hey there - yeah, just threw them in a repo:  https://github.com/bbatchelder/cura-dremel-3d20marlin

**Reply by ShinNinth** [reply]
Thank you I appreciate it!
